_posts/2019-12-26-policy-gradients.md

```diff
@@ -0,0 +1,136 @@
+---
+layout: post
+title:  "Understanding Policy Gradients"
+date:   2019-12-26 22:19:00 +0530
+categories: reinforcement-learning, policy-gradients
+---
+
+# Understanding Policy Gradients
+In reinforcement learning, we have an agent interacting with the environment. The agent acts, the environment gives feedback in the for of rewards. The agent wants to maximize the total rewards it gets.
+
```

Comment:
"in the form of rewards"? I assume that is just a typo.

Reply:
Corrected now